sharepoint/sharepoint-ps/sharepoint-online/Copy-SPOPersonalSitePage.md

```diff
@@ -0,0 +1,212 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/copy-spopersonalsitepage
+applicable: SharePoint Online
+title: Copy-SPOPersonalSitePage
+schema: 2.0.0
+author: xuyangzo, bellesirapha
+ms.author: xuyangzou, spodeanu
```

Comment:
I believe this needs to be 1 alias in each case -- it doesn't work for multiples:

```
Line 0, Column 0: [Warning: author-not-found - See documentation] Invalid value for author: 'xuyangzo, bellesirapha' is not a valid GitHub ID.

Line 0, Column 0: [Suggestion: ms-author-invalid - See documentation] Invalid value for 'ms.author', 'xuyangzou, spodeanu' is not a valid Microsoft alias.
```

Reply:
Ok I'm keeping Xuyang's on both. My assumption is this may get changed to the writer later.